python/dgl/contrib/sampling/sampler.py

```diff
@@ -240,6 +252,7 @@ def NeighborSampler(g, batch_size, expand_factor, num_hops=1,
         * str: indicates some common ways of calculating the number of sampled neighbors,
           e.g., ``sqrt(deg)``.
 
+        TODO(minjie): What will happen if expand_factor > num_neighbors?
```

Comment:
it'll sample all neighbors. this is how `create_full_node_flow` is implemented.

Reply:
Done.